src/FunderMaps.Webservice/Startup.cs

```diff
@@ -80,7 +80,7 @@ public void ConfigureServices(IServiceCollection services)
             {
                 // FUTURE: The full enum description support for swagger with System.Text.Json is a WIP. This is a custom tempfix.
                 c.SchemaFilter<EnumSchemaFilter>();
-                c.GeneratePolymorphicSchemas();
+                c.GeneratePolymorphicSchemas(); // TODO Obsolete
```

Comment:
I wrote the same comment in another branch

Reply:
We shall see my friend, we shall see